2021-11-23 13:42:06,458 INFO    MainThread:36282 [wandb_setup.py:_flush():69] setting env: {'project': 'genre_classification_production', 'run_group': 'dev'}
2021-11-23 13:42:06,458 WARNING MainThread:36282 [wandb_setup.py:_flush():69] could not save program above cwd: /Users/ahsanujundai/opt/anaconda3/envs/mlflow-7398531e5288219e428182dc4d6601ae57991d39/bin/pytest
2021-11-23 13:42:06,458 INFO    MainThread:36282 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-23 13:42:06,458 INFO    MainThread:36282 [wandb_init.py:_log_setup():334] Logging user logs to /Users/ahsanujundai/workspace/genre_classification/check_data/wandb/run-20211123_134206-2m5cpg5i/logs/debug.log
2021-11-23 13:42:06,458 INFO    MainThread:36282 [wandb_init.py:_log_setup():335] Logging internal logs to /Users/ahsanujundai/workspace/genre_classification/check_data/wandb/run-20211123_134206-2m5cpg5i/logs/debug-internal.log
2021-11-23 13:42:06,459 INFO    MainThread:36282 [wandb_init.py:init():367] calling init triggers
2021-11-23 13:42:06,459 INFO    MainThread:36282 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-11-23 13:42:06,459 INFO    MainThread:36282 [wandb_init.py:init():416] starting backend
2021-11-23 13:42:06,459 INFO    MainThread:36282 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-11-23 13:42:06,490 INFO    MainThread:36282 [backend.py:ensure_launched():123] starting backend process...
2021-11-23 13:42:06,498 INFO    MainThread:36282 [backend.py:ensure_launched():127] started backend process with pid: 36293
2021-11-23 13:42:06,501 INFO    MainThread:36282 [wandb_init.py:init():421] backend started and connected
2021-11-23 13:42:06,505 INFO    MainThread:36282 [wandb_init.py:init():461] updated telemetry
2021-11-23 13:42:06,506 INFO    MainThread:36282 [wandb_init.py:init():480] communicating current version
2021-11-23 13:42:07,392 INFO    MainThread:36282 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-23 13:42:07,392 INFO    MainThread:36282 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-11-23 13:42:07,798 INFO    MainThread:36282 [wandb_init.py:init():518] starting run threads in backend
2021-11-23 13:42:07,856 INFO    MainThread:36282 [wandb_run.py:_console_start():1476] atexit reg
2021-11-23 13:42:07,857 INFO    MainThread:36282 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2021-11-23 13:42:07,857 INFO    MainThread:36282 [wandb_run.py:_redirect():1344] Redirecting console.
2021-11-23 13:42:07,857 INFO    MainThread:36282 [redirect.py:install():213] install start
2021-11-23 13:42:07,858 INFO    MainThread:36282 [redirect.py:install():228] install stop
2021-11-23 13:42:07,858 INFO    MainThread:36282 [redirect.py:install():213] install start
2021-11-23 13:42:07,859 INFO    MainThread:36282 [redirect.py:install():228] install stop
2021-11-23 13:42:07,859 INFO    MainThread:36282 [wandb_run.py:_redirect():1390] Redirects installed.
2021-11-23 13:42:07,859 INFO    MainThread:36282 [wandb_init.py:init():542] run started, returning control to user process
2021-11-23 13:42:12,551 INFO    MainThread:36282 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-11-23 13:42:12,552 INFO    MainThread:36282 [wandb_run.py:_restore():1418] restore
2021-11-23 13:42:12,552 INFO    MainThread:36282 [redirect.py:uninstall():232] uninstall start
2021-11-23 13:42:12,552 INFO    MainThread:36282 [redirect.py:_stop():287] _stop: stdout
2021-11-23 13:42:12,552 INFO    MainThread:36282 [redirect.py:_stop():293] _stop closed: stdout
2021-11-23 13:42:12,553 INFO    stdout    :36282 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-11-23 13:42:12,553 INFO    stdout    :36282 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-11-23 13:42:12,553 INFO    MainThread:36282 [redirect.py:_stop():299] _stop joined: stdout
2021-11-23 13:42:12,553 INFO    MainThread:36282 [redirect.py:_stop():301] _stop rd closed: stdout
2021-11-23 13:42:12,553 INFO    MainThread:36282 [redirect.py:uninstall():236] uninstall done
2021-11-23 13:42:12,553 INFO    MainThread:36282 [redirect.py:uninstall():232] uninstall start
2021-11-23 13:42:12,553 INFO    MainThread:36282 [redirect.py:_stop():287] _stop: stderr
2021-11-23 13:42:12,553 INFO    MainThread:36282 [redirect.py:_stop():293] _stop closed: stderr
2021-11-23 13:42:12,554 INFO    stderr    :36282 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-11-23 13:42:12,554 INFO    stderr    :36282 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-11-23 13:42:12,554 INFO    MainThread:36282 [redirect.py:_stop():299] _stop joined: stderr
2021-11-23 13:42:12,554 INFO    MainThread:36282 [redirect.py:_stop():301] _stop rd closed: stderr
2021-11-23 13:42:12,554 INFO    MainThread:36282 [redirect.py:uninstall():236] uninstall done
2021-11-23 13:42:12,941 INFO    MainThread:36282 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 999
  total_bytes: 999
}

2021-11-23 13:42:14,948 INFO    MainThread:36282 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 999
  total_bytes: 3460
}

2021-11-23 13:42:16,955 INFO    MainThread:36282 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3460
  total_bytes: 3460
}

2021-11-23 13:42:18,081 INFO    MainThread:36282 [wandb_run.py:_show_files():1790] logging synced files
